### PR TITLE
added both html and css contents

### PR DESCRIPTION
**Pull request by Tekcoder**
Completed the structure of the website

### CODE REVIEW

**Review by stratospherique — COMMENTED** [review]
### Pull request under review by Ahmed Mahfoudh TSE

**Review by stratospherique — CHANGES_REQUESTED** [review]
### Great work :clap: However, some requirements needs to be implemented in order to get approval for your project. check below :arrow_down: 

#### Page structure

#### Navbar

image

- You have a undesirable visible border for the search input (it's overflowing). :pray: fix.

#### Bottom

* [ ] It contains four elements which are links (to another articles or `#`)
- Wrap each image inside a link and make it clickable.

#### General requirements:

- You linter is not properly configured. Check Microverse [guide](https://github.com/microverseinc/linters-config/tree/master/css) and follow the listed instructions.

- Add more info to your readme file (e.g. project descirption, technologies used, assignment link, your profile link, etc...)

### That's it :relieved: . When finished, kindly resubmit another code review  form :pray:

**Review by Redvanisation — CHANGES_REQUESTED** [review]
### Hello @Tekcoder 

**Good work on this project, but we need to fix the issues mentioned below before moving on ⬇️** 

1. You have a **white background on the search** icon in the navbar, please fix that.

2. Fix all the **linter's errors**.

### _After making the required changes please submit a new code review request_

### Good luck! 🍀

**Review by Forison — COMMENTED** [review]
# pull request under review by ADDO

**Review by Forison — CHANGES_REQUESTED** [review]
Hi @Tekcoder 
Thanks for addressing some of the issues addressed by the TSE, however, some issues still prevail.
here are a few changes we can make before merging.
- This does not look good
Screenshot from 2019-09-26 18-27-28
> fix this by wrapping by placing the navbar inside the section with class ```.hero```
> give the navbar a position attribute "relative" instead of absolute.

- the bottom does not look good
Screenshot from 2019-09-26 18-27-38
check the dimensions, to be precise their widths
>all four elements combined must have a width of 100%, hence each must have a width of 25%. with this they would align as they should

-  You should fix the warning given to you by the[ validator ](https://validator.w3.org)when uploading the index file.🙂
> you can use ```div```

#### Kindly fix these issues and submit for review 🙂
#### That is it @Tekcoder 🙂🙂
Happy Coding

File: style.css [hunk 1]
```diff
@@ -0,0 +1,131 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.main-navbar {
+  position: relative;
```
Comment (samgaco) [inline style.css:7]: :+1: 

[OPTIONAL]

What about improving this part a bit more so the navbar is more centered, and have some space left at the top? To simulate the original page in a better manner

**Review by samgaco — APPROVED** [review]
Hi :high_brightness: The project is approved :white_check_mark: you may merge to the master branch and submit the project completion form. Congratulations on finishing your first solo project :+1: 

-----------------

* ###### STATUS OF THE PROJECT: APPROVED :heavy_check_mark:

**Pull request merged**